articles/migrations/guides/extensibility-node12.md

```diff
@@ -0,0 +1,221 @@
+5. If you see an "Invalid API Key" error after updating, use the "Rotate" button to generate a new API key.
+
+![Authorization Extension Configuration](/media/articles/migrations/node-auth-ext-config.png)
+
+![Authorization Extension Buttons](/media/articles/migrations/node-auth-ext-buttons.png)
+
+### Delegated Administration URLs
+
+If you use the Delegated Administration Extension, the matrix that follows contains the updated URLs you must configure after you migrate to Node 12. The URL varies based on your location.
+
+| Location | Allowed Callback URL for Node 12 | Allowed Logout URL for Node 12 |
+| --- | --- | --- |
+| USA | `https://${account.tenant}.prod-us-node12.webtask.io/auth0-delegated-admin/login` | `https://${account.tenant}.prod-us-node12.webtask.io/auth0-delegated-admin` |
+| Europe | `https://${account.tenant}.prod-eu-node12.webtask.io/auth0-delegated-admin/login` | `https://${account.tenant}.prod-eu-node12.webtask.io/auth0-delegated-admin` |
+| Australia | `https://${account.tenant}.prod-au-node12.webtask.io/auth0-delegated-admin/login` | `https://${account.tenant}.prod-au-node12.webtask.io/auth0-delegated-admin` |
+
+For example, if you are located in the USA and you use the Delegated Administration, you should update the following fields in your application's settings: 
+- **Allowed Callback URLs**: `https://${account.tenant}.prod-us-node12.webtask.io/auth0-delegated-admin/login`
+- **Allowed Logout URLs**: `https://${account.tenant}.prod-us-node12.webtask.io/auth0-delegated-admin`
+
+### SSO Dashboard URLs
+
+The matrix that follows contains the updated URLs you must configure after you migrate to Node 12. The URL varies based on your location.
+
+The login URL for **Admins**:
+
+| Location | Allowed Callback URL |
+| --- | --- |
+| USA | `https://${account.tenant}.prod-us-node12.webtask.io/auth0-sso-dashboard/admins/login` |
+| Europe | `https://${account.tenant}.prod-eu-node12.webtask.io/auth0-sso-dashboard/admins/login` |
+| Australia | `https://${account.tenant}.prod-au-node12.webtask.io/auth0-sso-dashboard/admins/login` |
+
+The login URL for **Users**:
+
+| Location | Allowed Callback URL |
+| --- | --- |
+| USA | `https://${account.tenant}.prod-us-node12.webtask.io/auth0-sso-dashboard/login` |
+| Europe | `https://${account.tenant}.prod-eu-node12.webtask.io/auth0-sso-dashboard/login` |
+| Australia | `https://${account.tenant}.prod-au-node12.webtask.io/auth0-sso-dashboard/login` |
+
+### All Extensions
+
+Most extensions use the `PUBLIC_WT_URL` hidden secret for authorization. This secret depends on the runtime version and does not update automatically.
+
+To update it, you need to save the extension's settings (no changes are necessary). To do so, after switching the runtime to `Node 12`, you need to open the extension's settings in the extensions dashboard (gear icon) and hit `Save`. After that, the extensions gallery will update the `PUBLIC_WT_URL` secret accordingly based on the selected runtime.
```

Comment:
We verified that this process still works from Node 8 -> Node 12, so this section remains accurate. 👍